haystack/nodes/connector/crawler.py

```diff
@@ -94,15 +94,16 @@ def __init__(
                 options.add_argument("--no-sandbox")
                 options.add_argument("--disable-dev-shm-usage")
                 self.driver = webdriver.Chrome(service=Service("chromedriver"), options=options)
-            except:
-                raise Exception(
+            except Exception as e:
+                logger.warning(
```

Comment:
Why changing it into a warning? A failure to import the driver renders the crawler useless, so there's no point going ahead with the execution. Or have you seen cases in which it makes sense?

Reply:
The error is still raised below so this would still fail. And this makes sense because the raised error can be something else other than that the chrome driver is not found such as the `TypeError` that is shown in the issue.

Reply:
We could also explicitly catch the specific Exception where this error message makes sense.

Reply:
Omg I see it now :smile: I totally missed the `raise e` below. 

Anyway I'd go back to the exception and catch `ImportError` or `OSError`. have a look which one is raised when the driver is missing. We can use `raise from` to retain the original stacktrace.